packages/jest-utils/package.json

```diff
@@ -29,7 +29,7 @@
     "@types/jest": "^24.0.12 <= 24.0.13",
     "jest": "^24.7.1 <= 24.8.0",
     "ts-jest": "^24.0.2",
-    "typescript": "^3.4.5"
+    "typescript": "^3.8.3"
```

Comment:
The build was failing because some dependency of jest had declaration files incompatible with the previous 3.4.5 TS version. This should be a harmless change, but you are free to check it more in depth if you want